v2/tools/generator/internal/functions/index_registration_function.go

```diff
@@ -200,7 +200,8 @@ func (f *IndexRegistrationFunction) handleArray(result *dst.Ident, locals *astmo
 func (f *IndexRegistrationFunction) handleMap(result *dst.Ident, locals *astmodel.KnownLocalsSet, ident dst.Expr, remainingChain []*astmodel.PropertyDefinition) []dst.Stmt {
 	p := remainingChain[0]
 
-	key := locals.CreateLocal("key")
+	// key in the loop would always be un-used.
+	key := "_"
```

Comment:
Is this the first time we've used handle map? It seems weird to need to make this change if we've already used it, and wouldn't this change also impact across additional generated files? I don't see any in this PR.

Reply:
Yeah, that was weird for me.. I did check that a few times and did not find any usages.

Reply:
I'd expect our linters to catch anything the compiler missed, so no worries there. Seems weird is all.